.gitignore

```diff
@@ -1,6 +1,7 @@
 /target/
 # in case it's a symlink
 /target
+fedimint-client/target/
```

Comment:
This should no longer be needed. You might just have an old file from before https://github.com/fedimint/fedimint/pull/2286 . Or the new release didn't actually fix it. But they said they did.

Reply:
Didn't seem to fix it for me :frowning_face:

Reply:
Or maybe it did?  I'll remove it from this PR.